### PR TITLE
New Age Web3 Connection

### LINKED ISSUE #1: Inlude env reader for passing Infura Key
Currently we can invoke web3 to encode transfer data, but if (for example) we wanted to fetch token decimals, this would require passing an infura key to query the EVM.

This should be as easy as depending on `dotenv`, but doesn't appear to work as expected.

### PR DESCRIPTION
**Pull request by bh2smith**
Closes #1 by removing the need for it!

This eliminates the need for that gross old `connect.ts` file (Still needs to be removed)

We no longer have that missing INFURA KEY error and we no longer need any web3Provider.

web3Provider will be reintroduced in #5 (this is the reason for the leftover commented code).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@gnosis.pm/safe-apps-provider": "^0.3.0",
```
Comment (bh2smith) [inline package.json:16]: This new dependency isn't actually needed until the commented code is uncomented.

File: src/App.tsx [hunk 2]
```diff
@@ -10,15 +14,18 @@ import { parseCSV, Payment } from "./parser";
...
   const [transferContent, setTransferContent] = useState<Payment[]>([]);
   const [csvText, setCsvText] = useState<string>(
     "token_address,receiver,amount,decimals"
   );
-
   const { addMessage, setMessages } = useContext(MessageContext);
+  // const web3Provider = useMemo(
+  //   () => new ethers.providers.Web3Provider(new SafeAppProvider(safe, sdk)),
+  //   [sdk, safe]
+  // );
```
Comment (bh2smith) [inline src/App.tsx:28]: Gotta leave this stuff here, because otherwise I will forget. It took a bunch of digging to find this.

**Review by schmanu — APPROVED** [review]
Do I understand correctly why we dont need the Infuria Key anymore (until the next PR)?
We only used it for the encodeAbi() call of the transfer? And now we get the erc20 object without initializing a web3 connection?

Otherwise its a bit messy with the uncommented lines but for this transition period its fine. 

Nice work!

**Comment by bh2smith** [comment]
Yes that's right. There was never any real need for web3 to encode function calls. In the next PR when we fetch the token decimals from the EVM we will have to establish a web3 connection for this.

**Pull request merged**